### PR TITLE
Upgrade scalafmt to 1.2.0

### PR DESCRIPTION
**Pull request by densh**

### CODE REVIEW

**Comment by densh** [comment]
@olafurpg It looks like off-by-2 problems have been completely resolved in 1.2.0, thanks! 👍

File: nativelib/src/main/scala/scala/scalanative/native/CFunctionPtr.scala [hunk 1]
```diff
@@ -111,33 +111,22 @@ object CFunctionPtr {
...
-                                                                   T7,
-                                                                   T8,
-                                                                   T9,
-                                                                   T10,
-                                                                   T11,
-                                                                   T12,
-                                                                   T13,
-                                                                   R] =
+                              R](
+      f: Function13[T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, R])
```
Comment (olafurpg) [inline nativelib/src/main/scala/scala/scalanative/native/CFunctionPtr.scala]: 🙈

File: nir/src/main/scala/scala/scalanative/nir/Ops.scala [hunk 2]
```diff
@@ -46,15 +46,14 @@ object Op {
...
-  final case class Elem(ty: Type, ptr: Val, indexes: Seq[Val]) extends Pure
-  final case class Extract(aggr: Val, indexes: Seq[Int])       extends Pure
-  final case class Insert(aggr: Val, value: Val, indexes: Seq[Int])
-      extends Pure
-  final case class Stackalloc(ty: Type, n: Val)                   extends Op
-  final case class Bin(bin: nir.Bin, ty: Type, l: Val, r: Val)    extends Pure
-  final case class Comp(comp: nir.Comp, ty: Type, l: Val, r: Val) extends Pure
-  final case class Conv(conv: nir.Conv, ty: Type, value: Val)     extends Pure
-  final case class Select(cond: Val, thenv: Val, elsev: Val)      extends Pure
+  final case class Elem(ty: Type, ptr: Val, indexes: Seq[Val])      extends Pure
```
Comment (olafurpg) [inline nir/src/main/scala/scala/scalanative/nir/Ops.scala]: Enjoy that extra free character 😄

File: .scalafmt.conf [hunk 3]
```diff
@@ -5,3 +5,5 @@ project.excludeFilters = [
   scalalib/
 ]
 project.git = true
+maxColumn = 80
```
Comment (olafurpg) [inline .scalafmt.conf]: This is the default value

**Pull request merged**